shell/ci/release/docker-stitch.sh

```diff
@@ -45,9 +45,10 @@ stitch_and_push_image() {
   local image="$1"
   local remoteImageNames=()
   for imageRegistry in $imageRegistries; do
-    remoteImageNames+=("$(determine_remote_image_name "$APPNAME" "$imageRegistry" "$image")")
-    if [[ $will_push == "true" ]] && [[ $imageRegistry =~ amazonaws.com($|/) ]]; then
-      ensure_ecr_repository "$imageRegistry/$APPNAME"
+    local remoteImageName="$(determine_remote_image_name "$APPNAME" "$imageRegistry" "$image")"
```

Comment:
Suggestion: whoops,my in-brain linter isn't as thorough as the real one.
```suggestion
    local remoteImageName
    remoteImageName="$(determine_remote_image_name "$APPNAME" "$imageRegistry" "$image")"
```